roles/import-from-url/tasks/login.yml

```diff
@@ -1,6 +1,6 @@
 ---
 - name: Login As Super User
-  command: "oc login -u {{ admin_user }} -p {{ admin_password }}"
+  command: "oc login --insecure-skip-tls-verify=true -u {{ admin_user }} -p {{ admin_password }}"
```

Comment:
Any reason to skip tls verification? I worked for me with verification on.

Reply:
Hmm, it didn't work for me.  I guess it depends if you install via openshift-ansible or minishift/oc cluster up.